Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -430,8 +430,8 @@ extension VariableDeclSyntax {
 
         // Need to access pattern bindings to get name, type, and other info of a var decl
         let varmodels = self.bindings.compactMap { (v: PatternBindingSyntax) -> Model in
-            let name = v.pattern.firstToken(viewMode: .sourceAccurate)?.text ?? String.unknownVal
-            var typeName = ""
+            let name = v.pattern.trimmedDescription
+            var typeName = String.voidType
```

Comment:
[IMHO]

I would like to use other name for this `typeName` because `void` is sort of reserved word in Swift and this `typeName` is not same as `void` type in Swift.

I would like to suggest alternative naming such as `String.invalidType` or `String.unknownType` for VarDecl typeName.
How do you think of it?

Reply:
You are correct.
It's uncommon but not incorrect for the type name to not appear in variable decl.
Previously, these cases were implicitly treated as a `""` type but I've decided to eliminate this with this PR, so it should explicitly be represented as Optional.

I found similar instances in the `init()` as well, so I've made the same corrections there.

Reply:
I think representing them as Optional is better approach than previous one. Thank you!